axelor-stock/src/main/java/com/axelor/apps/stock/service/MassStockMoveService.java

```diff
@@ -0,0 +1,8 @@
+package com.axelor.apps.stock.service;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.stock.db.MassStockMove;
+
+public interface MassStockMoveService {
```

Comment:
this service will be injected in the repository. Also like other services, it must serve only one purpose, so please rename it `MassStockMoveSequenceService`